ArduSub/system.cpp

```diff
@@ -157,6 +157,8 @@ void Sub::init_ardupilot()
     g2.actuators.initialize_actuators();
 
     // flag that initialisation has completed
+    update_actuators_from_jsbuttons();
+    update_lights_from_rcin();
```

Comment:
Ideally this would happen before the lights outputs initialise, although from testing we did earlier I'm not sure if it's possible given the intertwined nature with the joystick button functions.

Reply:
yeah, I'm not sure this is worth worrying about, and the only symptom is a bad named_value_report until the light is actuated...